### ISSUE TITLE
Events<AppExit> broken

### ISSUE THREAD

**Issue by ashneverdawn** [bevyengine/bevy#608]
This has been broken since 0.2.0, it works in 0.1.3

```
use bevy::{app::ScheduleRunnerPlugin, app::AppExit, prelude::*};
use std::time::Duration;

fn main() {
    App::build()
      .add_plugin(ScheduleRunnerPlugin::run_loop(Duration::from_secs_f64(1.0)))
      .add_system(update.system())
      .run()
}

pub fn update(mut shutdown: ResMut<Events<AppExit>>) {
  println!("exiting...");
  shutdown.send(AppExit);
}
```

What should happen:
The app should exit

What happens:
The ScheduleRunnerPlugin stops controlling frame intervals, and the app doesn't exit.

**Comment by memoryruins** [comment]
Thanks for reporting! This is fixed on master branch #536

**Comment by ashneverdawn** [comment]
oh nice! I missed that. Thanks!
